app/scripts/modules/openstack/loadBalancer/configure/wizard/listeners.html

```diff
@@ -18,16 +17,14 @@
         <tr ng-repeat="listener in loadBalancer.listeners">
           <td><select class="form-control input-sm" ng-model="listener.externalProtocol"
```

Comment:
can we change listener.externalProtocol to listener.protocol?... to keep consistent with the label